Sort by last_edited_at on index page

christophwong opened the pull request:
This is a new field added to publishing-api, since the sorting and
writing of this field is handled by publishing-api, we just need to send
the correct params in our request for multiple content items on our
index page

Commit:
Sort by last_edited_at on index page

This is a new field added to publishing-api, since the sorting and
writing of this field is handled by publishing-api, we just need to send
the correct params in our request for multiple content items on our
index page

tuzz commented on spec/models/document_spec.rb:
```diff
@@ -33,12 +33,13 @@ def initialize(params = {})
             :base_path,
             :content_id,
             :updated_at,
```
Do we use `updated_at` for anything or can this be removed from the list of fields we request?

If so, we can also remove it from document's fields and make things simpler.

Commit:
Remove updated_at

Updated_at was used for ordering and is displayed on the index page. Now
that we switch to use `last_edited_at` there's no need to request for
that field anymore.
